src/sagas/problemsSaga.js

```diff
@@ -1,16 +1,19 @@
 import { call, put, takeLatest } from 'redux-saga/effects';
 
 import { getStatement } from 'apis/problemStatementApi';
-import { statementAction, statementActionFailed } from 'actions/problemStatementActions';
+import {
+  statementAction,
+  statementActionFailed,
+} from 'actions/problemStatementActions';
 import { PROBLEM_STATEMENT } from 'constants/actionConstants';
 
 // Problem statememt Saga
-export function* statementSaga() {
+export function* statementSaga(action) {
```

Comment:
Codacy has a fix for the issue: [Missing space before *.](https://app.codacy.com/gh/joshsoftware/OnCoT-Frontend/pullRequest?prid=7073195)
```suggestion
export function * statementSaga(action) {
```